ONSdigital/eq-author
# Eq 20 front end

**What**

Integrates front-end build into foundation django app

**How to test**
1) Check out the branch
2) Follow instructions in the README.md to build and deploy on ElasticBeanstalk
3) Visit the url and verify that the page loads correctly. (it should be blank)
4) Verify that the javascript resources have loaded correctly

**Who can test**

@dhilton I think this is one for you :)
